ArchiSteamFarm/Bot.cs

```diff
@@ -1173,8 +1173,21 @@ public sealed class Bot : IDisposable {
 			// We must escape our message prior to sending it
 			message = Escape(message);
 
-			for (int i = 0; i < message.Length; i += maxMessageLength) {
-				string messagePart = message.Substring(i, Math.Min(maxMessageLength, message.Length - i));
+			int i = 0;
+			int partLength;
+			while (i < message.Length) {
+				if (message.Length - i > maxMessageLength) {
+					int lastNewLine = message.LastIndexOf('\n', i+maxMessageLength, maxMessageLength);
+					if (lastNewLine > 0 && lastNewLine > i) {
```

Comment:
First condition is redundant, `i` can never be less than 0 here, therefore it should just be the second condition.